recipes/lgatr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lgatr" %}
+{% set version = "1.3.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lgatr-{{ version }}.tar.gz
+  sha256: dce13ade9e2cbd8f05c2ac008aaa1089f023784f65d80e546c2b95df45158f6e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=68
+    - wheel >=0.42
+    - pip
+  run:
+    - python {{ python_min }}
+    - pytorch >=2.0
+    - numpy
+    - einops
+    - opt_einsum
+
+test:
+  imports:
+    - lgatr
+  commands:
+    - pip check
+    - python -c "import lgatr; print(lgatr.__version__)"
```

Comment:
we typically consider this "redundant" as a test. if you remove this line i'll be able to merge.